### ISSUE TITLE
GPS. Activate position indicator

### ISSUE THREAD

**Issue by qmenen** [minorua/Qgis2threejs#107]
Mobile phone. Activate Position Indicator (3D) on the map as read from the device's gps. Any help about it?

**Comment by DigDigDig** [comment]
@qmenen have you tested your product on mobile?  I haven't found the plugin to be very mobile freindly.

**Comment by qmenen** [comment]
Yes, I usually try it on my cell phone (SamsungS3), and I'm happy with the results. I leave you two examples that work perfectly for me. (Patience with the download. My server is modest: http://www.cian.com/Proyecto/Lagos1.html. And, with greater resolution: http://www.cian.com/Proyecto/Lagos4.html.)
This intricate relief, Picos de Europa (Alpine orogeny, just 10 km from the Atlantic), in my region, Asturias (Spain), is a place I usually frequent. My project is to use qgis2threejs to offer mountain routes by taking advantage of the elegant 3D presentation of Minorua's work (and waiting for improvements in the typography presentation that version 2 promises ;-)). There is, as far as I know, no more reliable way of showing the traveler his situation in a context as abrupt as this. My idea is that the map can tell you what your position is at any given time. It is not difficult to represent it in 2D, but it escapes me how to implement it in. 3D of qgis2threjs (and I do not tell you to get the map to move in unison with you as in a Sygic) Thanks anyway for your suggestion. In it I am also.
